grafana/provisioning/dashboards/aligned/aggregator_batcher.json

```diff
@@ -1323,6 +1513,6 @@
   "timezone": "browser",
   "title": "Aggregator Data",
   "uid": "aggregator",
-  "version": 6,
+  "version": 2,
   "weekStart": ""
-}
+}
```

Comment:
I know that this is a generated Grafana file but I would add the EOL here

Reply:
Actually, my formatter was removing it. Thanks!